src/extension.ts

```diff
@@ -94,6 +129,142 @@ export class Extension {
         }
     }
 
+    private async _yesNoPickAsync(message: string): Promise<boolean | undefined> {
+        const input = await vscode.window.showQuickPick(['No', 'Yes'], { ignoreFocusOut: true, placeHolder: message });
+
+        return input === undefined ? undefined : input === 'Yes';
+    }
+
+    public async onCreateFiles(event: vscode.FileCreateEvent): Promise<void> {
+        const csproj = new CsprojWriter();
+        const files: string[] = [];
+        const projs: string[] = [];
+
+        for (let i = 0; i < event.files.length; i++) {
+            const file = event.files[i];
+            const proj = await csproj.getProjFilePath(file.fsPath);
+            const fileStat = await fs.lstat(file.fsPath);
+
+            if (proj !== undefined && !fileStat.isDirectory()) {
+                const alreadyInProj = await csproj.get(proj, file.fsPath) !== undefined;
+
+                if (!alreadyInProj) {
+                    files.push(file.fsPath);
+                    projs.push(proj);
+                }
+            }
+        }
+
+        if (files.length < 1) return;
+
+        const message = files.length > 1 ? 'You can choose build actions on the newly added files' : 'You can choose a build action on the newly added file';
+        const button = files.length > 1 ? 'Choose build actions' : 'Choose a build action';
+
+        await vscode.window.showInformationMessage(message, button).then(async event => {
+            if (!event) return;
```

Comment:
One time you write with { other without {.

Reply:
Same as above. Thank you